elrond-wasm-node/src/api/send_api_node.rs

```diff
@@ -310,12 +328,34 @@ impl SendApi<ArwenBigUint> for ArwenApiImpl {
 		// account-level built-in function, so the destination address is the contract itself
 		let own_address = self.get_sc_address();
 
-		self.execute_on_dest_context(
+		let _ = self.execute_on_dest_context(
 			gas,
 			&own_address,
 			&ArwenBigUint::from(0u32),
 			function,
 			&arg_buffer,
-		)
+		);
+	}
+}
+
+/// Retrieves already pushed results, via `finish`.
+unsafe fn get_return_data(return_index: i32) -> BoxedBytes {
```

Comment:
Suggest reordering the function definitions: `get_return_data_range()` comes first, and `get_return_data()` comes second.